{"name": "test_something", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'EnvData' has no attribute 'data'", "trace": "self = <TestCases.test.TestCase object at 0x000001F8ED1364C8>\n\n    def test_something(self):\n        print('执行测试')\n        # 获取前置方法中产生的数据\n        # 从全局变量中获取\n>       print(EnvData.data)\nE       AttributeError: type object 'EnvData' has no attribute 'data'\n\ntest.py:31: AttributeError"}, "start": 1647350993971, "stop": 1647350993971, "uuid": "84e86ed4-052f-4ca0-8339-0140d73fb467", "historyId": "39a634f11d7b84a6209b18be84273ba0", "testCaseId": "5c116e581817ad4319fbae74ed49d15d", "fullName": "TestCases.test.TestCase#test_something", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "DESKTOP-76566J4"}, {"name": "thread", "value": "9076-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test"}]}